Add documentation for System Configuration (#245)
NUbots/NUbook: src/book/03-guides/03-hardware/02-flashing.mdx

   - The installation script is located in the NUbots repository at [doc/ArchInstall/arch_install.sh](https://github.com/NUbots/NUbots/blob/main/doc/ArchInstall/arch_install.sh)

6. Execute the script and follow the instructions

   - ```bash
     ./arch_install.sh
     ```

7. The installation script will end by downloading a secondary script and providing you with a command that you must run

   ```bash
   ROBOT_NUMBER=<N> arch-chroot /mnt ./arch-chroot_install.sh
   ```

   Substitute `<N>` with the number of the robot that you are building. This will influence the IP address of the robot as well as the robots' hostname.

   <Alert type="warning">

   The secondary script assumes that the network interfaces are named `eth0` for ethernet and `wlp58s0` for WiFi. If you _know_ this is not the case then execute the script as follows

   ```bash
   ROBOT_NUMBER=<N> \
   ETHERNET_INTERFACE=IF_NAME \
   WIFI_INTERFACE=IF_NAME \
   arch-chroot /mnt ./arch-chroot_install.sh
   ```

   </Alert>

> 8. Once that script has finished, you must run one more command.

Reviewer: ```suggestion
8. Once that script has finished, run the following command.
```
